### ISSUE TITLE
Convert account into an organization or create a new organization and move the project to it

### ISSUE THREAD

**Issue by lauxjpn** [bubibubi/EntityFrameworkCore.Jet#62]
This is a follow-up to the discussion under #34.

@bubibubi Please also set aside some time in the near future (within the next two weeks or so), to either convert the GitHub `bubibubi` account into a GitHub organization (which would mean you would need to move unrelated projects to a new personal account) or to create a new GitHub organization and move `EntityFrameworkCore.Jet` to it (in which case a new AZDO organization should be created as well; you could keep using the old one then for your personal account). If you want to go the latter route, we should brainstorm some new organization name.

**Comment by lauxjpn** [comment]
@bubibubi Can we proceed with this over the course of this week?

/cc @ChrisJollyAU

**Comment by lauxjpn** [comment]
> @bubibubi Can we proceed with this over the course of this week?

@bubibubi Please confirm. Thank you.

**Comment by lauxjpn** [comment]
@bubibubi Since you didn't respond (but were active in a different comment on this repo) I will now move along with this as I see fit.

* I will pick a name and then create a new GitHub organization.
* I will then add @bubibubi, @ChrisJollyAU and @lauxjpn to the organization.
* We then transfer [bubibubi/EntityFrameworkCore.Jet](https://github.com/bubibubi/EntityFrameworkCore.Jet) to the new organization.
* There is no need for AZDO anymore, as we have prepared the switch to GitHub Actions.
* I will create a new myget.org feed to replace the AZDO feed.

This will happen over the course of the week and is tracked by this issue here.

**Comment by ChrisJollyAU** [comment]
@lauxjpn do you have the permissions to do the transfer?

Also, recommend changing the default merge behaviour to squash and merge

**Comment by lauxjpn** [comment]
> @lauxjpn do you have the permissions to do the transfer?

@ChrisJollyAU No, @bubibubi needs to do the transfer once the new organization is setup. Since this is a very simple process (see [Transferring a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-personal-account)), it should only take 5 minutes.

The only other option is to simply fork the repository to the new org (and probably update the current repo to manually point to the new repo). For the existing user base, the official transfer is a bit smoother, since the old repo should then redirect automatically to the new one. However, the fork option would be good enough as well.

> Also, recommend changing the default merge behaviour to squash and merge

Definitely!

**Comment by bubibubi** [comment]
So sorry guys that I can't help you.
I will create the new org and I transfer the repository within a couple of days.
And I also think that I will restart help you in mid 2024 that I will update my projects (that are still on EF 6, not EF Core).

**Comment by lauxjpn** [comment]
@bubibubi Great! Do it until Sunday. Otherwise, I will do it myself on Monday. If you have ideas for open source organization names (that are available), pitch them here first, so we can all decide on it.

**Comment by lauxjpn** [comment]
> Only suggestion I have at the moment is something like: CyrrusRed

I would be fine with that.

**Comment by ChrisJollyAU** [comment]
> > Only suggestion I have at the moment is something like: CyrrusRed
> 
> I would be fine with that. (Would also implicitly provide us with a primary color and maybe even a shape for things like a logo etc.)

Actually, just realised I misspelt it. Should be CirrusRed.

MS Access originally went by the codename of Cirrus and the JET Database engine was also referred to as Jet Red. (Therre was a Jet Blue that was meant to supersede it, but it became the Extensible Storage Engine used in Exchange and Windows Search)

**Comment by bubibubi** [comment]
Ok for CirrusRed then!!!
Don't ask me to draw a logo. I draw even worst than I write code 😆 
I wait till tomorrow afternoon GMT just to be sure.

Should I also move the provider for EF 6?
I'm considering it at end of life.

**Comment by lauxjpn** [comment]
Great, its `CirrusRed` then.

Logo can come later 😄 

> Should I also move the provider for EF 6?

Sounds good. I think anything topic related makes sense.

**Comment by bubibubi** [comment]
Bad news... CirrusRed is already taken.
We need other ideas...

**Comment by ChrisJollyAU** [comment]
> Bad news... CirrusRed is already taken.

On no.

- RedCirrus
- JetRed

**Comment by lauxjpn** [comment]
`JetRed` would be fine too.
Otherwise, `CirrusBlue`?

**Comment by ChrisJollyAU** [comment]
The only reason why the colour was red, was because the version of JET for mdb was called Jet Red. Got to think of other suggestions now

**Comment by bubibubi** [comment]
So guys, other ideas?
* CirrusBlue
* CirrusRedNet (or some other options)
* ???

**Comment by ChrisJollyAU** [comment]
Could add a dash somewhere
- Cirrus-Red
- CirrusRed-Net  (might be the best option. don't think we can use the . in it so the dash is probably next best)

**Comment by lauxjpn** [comment]
`CirrusRed-Net` and `CirrusRedNet` is both fine with me.

(For namespaces and places where dashes are not allowed, `CirrusRed-Net` would implicitly become `CirrusRedNet`.)

**Comment by bubibubi** [comment]
My preferred are
* CirrusRedOrg
* Cirrus-Red 

The domains cirrusred.org and cirrus-red.org are both available.

@ChrisJollyAU , we need only your ok!

**Comment by ChrisJollyAU** [comment]
CirrusRedOrg should be fine

**Comment by lauxjpn** [comment]
It's `CirrusRedOrg` then.

**Comment by ChrisJollyAU** [comment]
Can bring that in if you want. It is pretty much historical by now so might want to set it to archived status. Otherwise can just append `.EF6` to its name

@lauxjpn Have disable the option to create a merge commit and added a protection rule to the master branch requiring linear history

**Comment by lauxjpn** [comment]
I'll update the CI later today.

**Comment by lauxjpn** [comment]
I sent out a couple of invitations for the new myget.org feeds.

I also created an organization account for nuget.org and made it one of the owners of the existing packages.
@bubibubi I added you as an admin to the new nuget.org organization.
@ChrisJollyAU If you post (or send me) your nuget.org username, I'll do the same for your account.

The nuget.org organization required an internal email address, so I created a dedicated one. If you want emails forwarded from that email address to your personal ones, just say so and I'll put them in as forward targets.

**Comment by bubibubi** [comment]
**LAST STEPS**
1. I've just bought the domain cirrusred.org. If we'd like to do something the domain is our.
2. We should also update the EF Core providers documentation. https://learn.microsoft.com/en-us/ef/core/providers/?tabs=dotnet-core-cli

**Comment by ChrisJollyAU** [comment]
I did have it on my mental to do list to get the compatible version updated (was aiming for when v8 was out)

**Comment by lauxjpn** [comment]
> 1. I've just bought the domain cirrusred.org. If we'd like to do something the domain is our.

@bubibubi Just let it redirect to this repo for now.

**Comment by ChrisJollyAU** [comment]
Anything more to do on this topic?
@lauxjpn @bubibubi

**Comment by lauxjpn** [comment]
> Anything more to do on this topic?

I think open are still:
* Redirecting requests that go to `cirrusred.org` to this repo for now (@bubibubi I can provide you with authoritative DNS servers and handle the redirect if you want me to.)
* Updating the [Database Providers](https://learn.microsoft.com/en-us/ef/core/providers/) EF Core docs with the new repo name/link (maybe @ajcvickers or @roji can help)

**Comment by roji** [comment]
@lauxjpn sure thing - can one of you guys submit a PR? Usually the maintainers of the providers are the one doing the updates (they know best what's going on)

**Comment by ChrisJollyAU** [comment]
Ok. have done a PR ([EntityFrameworkCore.Docs#4561)](https://github.com/dotnet/EntityFramework.Docs/pull/4561)
